### PR TITLE
Add logrotate for docker

### LINKED ISSUE #552: Log rotation for docker logs
It appears that docker logs can grow quite large (800MB after a couple of days) and cause issues with disk space on the boot volume:

```
$ ll /var/log/docker
-rw-r--r-- 1 root root 836051643 Apr 18 15:26 /var/log/docker
```

This file was mainly full of similar logs:

```
$ tail /var/log/docker
2016/04/18 16:02:23 http: multiple response.WriteHeader calls
time="2016-04-18T16:02:23.518656414Z" level=info msg="GET /containers/44aebbd65a08570b33ec51aac9a4e01a744511f80158c70edfc7cd3671758676/json"
time="2016-04-18T16:02:23.519068975Z" level=info msg="GET /containers/4fc1b1758895dd73d75c6b8112479310931979a9754c4346b26653720ee58a06/top"
time="2016-04-18T16:02:23.527493833Z" level=info msg="GET /containers/4fc1b1758895dd73d75c6b8112479310931979a9754c4346b26653720ee58a06/json"
time="2016-04-18T16:02:23.528072817Z" level=info msg="GET /containers/9e3f44d19d170a099ccdceec9dcaab59ebd3d79e5118a30d06e303b1eeb62916/top"
time="2016-04-18T16:02:23.539116829Z" level=info msg="GET /containers/9e3f44d19d170a099ccdceec9dcaab59ebd3d79e5118a30d06e303b1eeb62916/json"
```

### PR DESCRIPTION
**Pull request by mwarkentin**
Fixes #552 - untested so far.

* [logrotate docs](http://www.linuxcommand.org/man_pages/logrotate8.html)

## Release Playbook
- [ ] Rebase against master
- [ ] Release branch ()
- [ ] Pass CI ()
- [ ] Code review
- [ ] Merge into master
- [ ] Release master ()
- [ ] Pass CI ()
- [ ] Update staging rack
- [ ] Edit [Rack release record](https://github.com/convox/rack/releases) and/or update [docs](https://github.com/convox/convox.github.io)
- [ ] Publish release
- [ ] Release CLI

Add a simple logrotate configuration for `/var/log/docker` and `/var/lib/docker/containers/*/*.log`

Rotates logs daily, and keeps 1 week worth.

### CODE REVIEW

File: api/dist/kernel.json [hunk 1]
```diff
@@ -861,6 +861,8 @@
             "echo 'exclude=kernel*' >> /etc/yum.conf",
             "yum -y update",
             "yum -y install aws-cfn-bootstrap",
+            "echo '/var/log/docker {\n  rotate 7\n  daily\n  nocompress\n  copytruncate\n}\n' >> /etc/logrotate.d/docker",
+            "echo '/var/lib/docker/containers/*/*.log {\n  rotate 7\n  daily\n  nocompress\n  copytruncate\n  missingok\n}\n' >> /etc/logrotate.d/docker",
```
Comment (robatwave) [inline api/dist/kernel.json]: pretty sure it should be 'echo -e' for the \n to work

**Comment by nzoschke** [comment]
Thanks! The logrotate.d strategy looks good on first glance. I'll help test.

Docker logs should already be rotated based on the custom args we pass to the daemon: 

```
"echo 'OPTIONS=\"${OPTIONS} --log-opt max-file=2 --log-opt max-size=50m --host=unix:///var/run/docker.sock  --host=0.0.0.0:2376\"' >> /etc/sysconfig/docker"
```

**Comment by mwarkentin** [comment]
@nzoschke Updated to remove the container logs.

**Comment by mwarkentin** [comment]
For testing, we can use [`logrotate -f`](http://explainshell.com/explain?cmd=logrotate+-f) to force a rotation.

**Comment by nzoschke** [comment]
Closing this since I copied it onto https://github.com/convox/rack/pull/588

Thanks for the spike.

**Pull request closed without merging**